ModularTegustation/tegu_traitor_code/tegu_traitor_items.dm

```diff
@@ -157,6 +157,14 @@
 	var/datum/component/storage/STR = GetComponent(/datum/component/storage)
 	STR.max_items = 10
 
+/obj/item/melee/sabre/breadstick
```

Comment:
Why is this put right stack in the middle of the traitor viro kit code?

Reply:
That would be my mistake in not reading enough of the code around it.